### ISSUE TITLE
macOS 14.3 `multipass launch docker` raises "launch failed: '24.04' is not a supported alias"

### ISSUE THREAD

**Issue by Cam2337** [canonical/multipass-blueprints#53]
```
ctew@ctew-mbp /Users/ctew % multipass launch -n manager docker
launch failed: '24.04' is not a supported alias. Please use `multipass find` for supported image aliases.
```

Running `multipass find` lists the `24.04` image:
```
ctew@ctew-mbp /Users/ctew % multipass find
Image                       Aliases           Version          Description
20.04                       focal             20240513         Ubuntu 20.04 LTS
22.04                       jammy             20240514         Ubuntu 22.04 LTS
23.10                       mantic            20240514         Ubuntu 23.10
24.04                       noble,lts         20240523.1       Ubuntu 24.04 LTS

Blueprint                   Aliases           Version          Description
anbox-cloud-appliance                         latest           Anbox Cloud Appliance
charm-dev                                     latest           A development and testing environment for charmers
docker                                        0.5              A Docker environment with Portainer and related tools
jellyfin                                      latest           Jellyfin is a Free Software Media System that puts you in control of managing and streaming your media.
minikube                                      latest           minikube is local Kubernetes
ros-noetic                                    0.1              A development and testing environment for ROS Noetic.
ros2-humble                                   0.1              A development and testing environment for ROS 2 Humble.
```

Potentially related to #50 or #51?

**Comment by Cam2337** [comment]
```
ctew@ctew-mbp /Users/ctew % multipass --version
multipass   1.13.1+mac
multipassd  1.13.1+mac
```

**Comment by sharder996** [comment]
Ah, this is because the docker blueprint got updated to use 24.04, but Multipass doesn't recognize the 24.04 alias yet. This will be fixed in version 1.14.

@ricab Do we want to revert the change to the docker blueprint in the meantime?

**Comment by ricab** [comment]
Ouch, indeed let's revert.
